config.go

```diff
@@ -155,37 +124,37 @@ func (c Config) getGraylogTLSTimeout() time.Duration {
 	return time.Second * time.Duration(timeoutSeconds)
 }
 
-func (c Config) getGraylogLogEnvName() string {
+func (e *EnvConfig) getGraylogLogEnvName() string {
 	// Check if we already memoized the log env name.
-	if c.logEnvName != nil {
-		return *c.logEnvName
-	}
-
-	env, err := c.getGraylogEnv()
-	if err != nil {
-		panic(err)
-	}
-
-	var logEnvName string
-	if env == prodEnv {
-		logEnvName = "prd"
+	if e.logEnvName != nil {
```

Comment:
Any reason we need to memoize? How often is this method called?

Reply:
I think I changed the initialize implementation so I think it's actually only called once, so I can remove the memoization here.